app/assets/scripts/components/documents/document-download-menu.js

```diff
@@ -106,7 +106,7 @@ export default function DocumentDownloadMenu(props) {
 
           setTimeout(() => {
             fetchPdf(`${pdfUrl}?retry=true`);
-          }, 3000);
+          }, 30000);
```

Comment:
```suggestion
          }, 10000);
```

Reply:
May be we should consider turning this into a constant as well - something like `initialWait`.